### ISSUE TITLE
Separation Project

### ISSUE THREAD

**Issue by Kasualix** [MCTeamPotato/Opotato#6]
As you can see, Opotato is a massive mod with tons of features and configuration for Minecraft 1.16.5.
And obviously, some people probably only need one feature of it, but after downloading this mod, they have too many features they don't need at all.
Do you want me to split up the features of Opotato?
Which feature do you want?
Simply write it down here, and I will do it.

**Comment by Kasualix** [comment]
Roadmap 预留位

**Comment by anoomolu** [comment]
It might be nice to have the Alternate Current feature split off into its own mod, for me personally that’s the one feature from opotato I use it for.

**Comment by Kasualix** [comment]
Oh, I think I need to give some more info.
1. If you won't uninstall Opotato and install the split mod after the mod is split, don't mention it here.
2. This splitting plan don't mean that Opotato will remove its features.

**Comment by Kasualix** [comment]
> It might be nice to have the Alternate Current feature split off into its own mod, for me personally that’s the one feature from opotato I use it for.

Do you insist on this after these two info? If so, I will do it this weekend.

**Comment by anoomolu** [comment]
> > It might be nice to have the Alternate Current feature split off into its own mod, for me personally that’s the one feature from opotato I use it for.
> 
> Do you insist on this with these two rules? If so, I will do it this weekend.

Yep I understand, that's fine. Thanks for making a really useful mod either way, love for 1.16 is always appreciated :)

**Comment by aarter180** [comment]
A mod that only has the fix for the structure value error in cataclysm would be neat.

**Comment by Kasualix** [comment]
Alternate Current Potato Edition: https://github.com/MCTeamPotato/PotatoCurrent/releases/tag/0.0.1
Cataclysm Fixer: https://github.com/MCTeamPotato/CataclysmFixer/releases/tag/0.0.1
I will upload them to CurseForge when I have time

**Comment by SSSlavic** [comment]
Disable Internet connection behavior with more mod support, for instance abnormal core/blueprint

**Comment by Kasualix** [comment]
> Disable Internet connection behavior with more mod support, for instance abnormal core/blueprint

This is off-topic for this issue, you do should create a new one.
But I will  do this in the next release anyway.

**Comment by anoomolu** [comment]
Thanks for taking the time to upload Potatocurrent ^^

Would you be able to make the Blueprint/Abnormals Core internet disabling feature a standalone mod? Many mods outside of Abnormals' rely on this library (my use case is Compat O' Plenty).

**Comment by Kasualix** [comment]
Oh, sorry for the late reply, my github notification seems broken for some weird reason, I will do it tonight.

**Comment by Kasualix** [comment]
Done. waiting for moderator approval
https://legacy.curseforge.com/minecraft/mc-mods/blueprintinternetconnectiondisabler

**Comment by TexBlock** [comment]
Mixin in Heaven?

**Comment by Kasualix** [comment]
https://legacy.curseforge.com/minecraft/mc-mods/mixin-in-heaven
Done. waiting for approval

**Comment by SSSlavic** [comment]
Quark module would be good

**Comment by Kasualix** [comment]
https://legacy.curseforge.com/minecraft/mc-mods/quark-patcher
done.
